plugins/dapper/README.md

```diff
@@ -0,0 +1,168 @@
+# Dapper Plugin for SBOM Surfactant
+
+A plugin for Surfactant that uses [Dapper](https://github.com/LLNL/dapper) to identify software packages from binary files by querying comprehensive package databases.
+
+## Overview
+
+The Dapper plugin enhances Surfactant's SBOM generation by identifying which software packages provide specific binary files. It queries multiple Linux distribution databases (Ubuntu, Debian) to find package information for ELF files, with planned support for Windows PE files via NuGet databases.
+
+## Features
+
+- **Multi-distribution package detection**: Queries all available Linux distributions (Ubuntu Focal/Jammy/Noble, Debian Buster/Bullseye/Bookworm)
+- **Comprehensive results**: Returns all matching packages across distributions
+- **File normalization**: Handles various file naming conventions through intelligent normalization
+- **Structured output**: Provides detailed package information including package names, versions, and source distributions
+
+## Prerequisites
+
+### Install Dapper and Datasets
+
+Before using this plugin, you need to install Dapper and download the package databases:
+
+1. Install dapper-python (required for database queries):
+   ```bash
+   pip install dapper-python
+   ```
```

Comment:
If dependencies were listed properly in pyproject.toml for the plugin, this step should be removed.